mattermost-plugin/go.mod

```diff
@@ -4,10 +4,12 @@ go 1.12
 
 replace github.com/mattermost/focalboard/server => ../server
 
+replace github.com/mattermost/mattermost-plugin-api/v6 => ../../../../../../dev/mattermost-plugin-api
```

Comment:
This `replace` exists for testing purposes. Once the `mattermost-plugin-api` v6 PR gets merged, the dependency will be updated and the `replace` removed.